### PR TITLE
Add 11.35 release notes

### PR DESCRIPTION
**Pull request by tas50**
#### Description

New week. New release

#### Types of changes

- [ ] Functional documentation bug fix (i.e., broken link or some other busted behavior)
- [ ] New functional doc capabilities (i.e., filter search results)
- [x] New content
- [ ] Revision to existing content
- [x] Chore (non-breaking change that does not add functionality or fix an issue)

#### Checklist

- [x] I have read the **README** document about contributing to this repo.
- [x] I have tested my changes locally and there are no issues.
- [x] All commits are signed.

### CODE REVIEW

File: releases/2024-12-17-mondoo-11.35-is-out.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+## 🎉 NEW FEATURES
+
+### Microsoft Defender for Cloud findings in Mondoo
+
+- https://github.com/mondoohq/console/pull/5270
+- https://github.com/mondoohq/console/pull/5238
+
+### Network security with Nmap scanning
+
+Nmap, short for Network Mapper, is a powerful and versatile open-source tool used for network discovery and security auditing. This tool is widely utilized by network administrators, security professionals, and penetration testers to map out network structures, discover hosts, identify services, and detect vulnerabilities. Now you can integrate Nmap results with Mondoo findings using our new cnquery/cnspec Nmap provider.
```
Comment (misterpantz) [inline releases/2024-12-17-mondoo-11.35-is-out.md:25]:
```suggestion
Nmap, short for Network Mapper, is a powerful and versatile open source tool used for network discovery and security auditing. Network administrators, security professionals, and penetration testers rely on Nmap to map out network structures, discover hosts, identify services, and detect vulnerabilities. Now you can integrate Nmap results with Mondoo findings using our new cnquery/cnspec Nmap provider.
```

File: releases/2024-12-17-mondoo-11.35-is-out.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+
+Scan active IP address in network:
```
Comment (misterpantz) [inline releases/2024-12-17-mondoo-11.35-is-out.md:27]:
```suggestion
Scan active IP address in a network:
```
??

File: releases/2024-12-17-mondoo-11.35-is-out.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+        port: 7000
+        product: "AirTunes rtspd"
+      }
+    ]
+    name: "192.168.178.25"
+  }
+]
+```
+
+Scan specific host IP:
```
Comment (misterpantz) [inline releases/2024-12-17-mondoo-11.35-is-out.md:81]:
```suggestion
Scan a specific host IP:
```

File: releases/2024-12-17-mondoo-11.35-is-out.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+No one wants to wait so we made Mondoo faster than ever.
+
+- 4x faster compliance report generation
+- 7x faster asset discovery during large scans
+- 20x faster GitHub repository discovery in large organizations
+- Reduced GitHub API call usage during organization scans
+
+### New checks in Mondoo DNS Security
+
+The Mondoo DNS Security policy now includes full descriptions and impact scores for each check as well as new checks to ensure DNSSEC is enabled and no DNS wildcard entries exist.
```
Comment (misterpantz) [inline releases/2024-12-17-mondoo-11.35-is-out.md:108]:
```suggestion
The Mondoo DNS Security policy now includes full descriptions and impact scores for each check. There are also new checks to ensure DNSSEC is enabled and no DNS wildcard entries exist.
```

File: releases/2024-12-17-mondoo-11.35-is-out.md [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+
+### Display remediated asset count on CVEs
+
+See not just the current state of CVEs, but the work you've done to resolve findings with remediated counts on CVEs and advisory pages.
```
Comment (misterpantz) [inline releases/2024-12-17-mondoo-11.35-is-out.md:112]:
```suggestion
It's important to observe not just the current state of CVEs, but also the work you've done to resolve them. Remediated counts on CVE and advisory pages provide the full scope of information.
```

File: releases/2024-12-17-mondoo-11.35-is-out.md [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+- Deprecate `createdTime` in favor of a new `createdAt` field
+
+#### github.organization
+
+- New `totalPublicRepos` field
+
+## 🐛 BUG FIXES AND UPDATES
+
+- Fix affected assets lists not containing assets scanned by older custom policies.
+- Show the link to discovered assets in each integration page.
```
Comment (misterpantz) [inline releases/2024-12-17-mondoo-11.35-is-out.md:160]:
```suggestion
- Show the link to discovered assets on each integration page.
```

**Review by misterpantz — APPROVED** [review]

File: releases/2024-12-17-mondoo-11.35-is-out.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+
+### Microsoft Defender for Cloud findings in Mondoo
+
+Mondoo now imports vulnerabilities discovered by Microsoft Defender for Cloud, so you can prioritize them alongside vulnerabilities discovered by Mondoo. Mondoo helps security teams quickly make sense of security findings regardless of where those findings originate, prioritize the most critical risks, and easily assign them to IT and DevOps with detailed remediation steps. This means that Mondoo significantly speeds up time to resolution and bolsters overall security posture.
+
+Vulnerabilities discovered by Microsoft Defender for Cloud appear in the Mondoo console alongside those discovered by Mondoo’s own scans. Mondoo automatically enriches all vulnerabilities found by Microsoft Defender with additional risk data and remediation advice, and prioritizes them based on the actual risk posed in the environment.
+
+Mondoo enriches the findings generated by Microsoft Defender for Cloud with severity, known exploit data, exploit prediction scoring, asset exposure, and business impact, to make sure your security teams are focused on resolving the most important problems.
+
+
```
Comment (tas50) [inline releases/2024-12-17-mondoo-11.35-is-out.md:26]:
```suggestion
```

**Pull request merged**